Community Docs (#2069)
fishtown-analytics/docs.getdbt.com: website/docs/community/contribute.md

[The dbt Community](https://www.getdbt.com/community/) exists to allow analytics practitioners to share their knowledge, help others and collectively to drive forward the discipline of analytics engineering. This is something that can’t be done by any one individual or any one organization - to create a new discipline is necessarily a community effort. The **only** reason that dbt has become as widespread as it has is because people like you choose to get involved and share your knowledge. Contributing to the community can also be a great way to learn new skills, build up a public portfolio and make friends with other practitioners.

There are opportunities here for everyone to get involved. Whether you are just beginning your analytics engineering journey or you are a seasoned data professional - contributing isn’t about knowing all of the answers, it’s about learning things together.

Below - you’ll find a sampling of the ways to get involved. It might feel complicated or like there are a lot of options - but these are ultimately just variations on the theme of sharing knowledge with the broader community.

> <DocCardList items={useCurrentSidebarCategory().items} />

Reviewer: is this basically a way to do an auto-generated "category" page but actually have a more fleshed-out Markdown intro rather than just a simple string?

Author: yep exactly! took some digging to figure it out but that's precisely it.

---

> title: Become a contributor

Reviewer 2: I (and the Docs team) deeply dislike having pages which are only accessible by clicking the header, especially when Additional resources doesn't behave the same way and just toggles expanded/contracted state
![image](https://user-images.githubusercontent.com/7335046/194965738-a400816b-9e87-45ea-bcbb-749b3ee215f0.png)

Could this instead switch to be a fifth page called `Become a Contributor`?

Author: strong disagree! but i'm not that invested in it, so sure

id: contribute
---

<section className="community-home">

## Want to get involved? Start here

The dbt Community predates dbt Labs as an organization and harkens back to the days when a scrappy analytics consultancy of a few [pissed off data analysts](<https://www.hashpath.com/2020/12/an-analytics-engineer-is-really-just-a-pissed-off-data-analyst/#:~:text=Often%20times%2C%20an%20analytics%20engineer,necessity%20(and%20genius%20branding).>) started hacking together an open source project around which gathered a community that would change how the world uses data.

> [The dbt Community](https://www.getdbt.com/community/) exists to allow analytics practitioners to share their knowledge, help others and collectively to drive forward the discipline of analytics engineering. This is something that can’t be done by any one individual or any one organization - to create a new discipline is necessarily a community effort. The **only** reason that dbt has become as widespread as it has is because people like you choose to get involved and share your knowledge. Contributing to the community can also be a great way to learn new skills, build up a public portfolio and make friends with other practitioners.

Reviewer 3: ```suggestion
The dbt Community exists to allow analytics practitioners to share their knowledge, help others and collectively to drive forward the discipline of analytics engineering. This is something that can’t be done by any one individual or any one organization - to create a new discipline is necessarily a community effort. The only reason that dbt has become as widespread as it has is because people like you choose to get involved and share your knowledge. Contributing to the community can also be a great way to learn new skills, build up a public portfolio and make friends with other practitioners.
```

> There are opportunities here for everyone to get involved. Whether you are just beginning your analytics engineering journey or you are a seasoned data professional - contributing isn’t about knowing all of the answers, it’s about learning things together.

Reviewer 3: ```suggestion
There are opportunities here for everyone to get involved, whether you are just beginning your analytics engineering journey or you are a seasoned data professional. Contributing isn’t about knowing all of the answers, it’s about learning things together.
```

> Below - you’ll find a sampling of the ways to get involved. It might feel complicated or like there are a lot of options - but these are ultimately just variations on the theme of sharing knowledge with the broader community.

Reviewer 3: ```suggestion
Below you’ll find a sampling of the ways to get involved. There are a lot of options but these are ultimately just variations on the theme of sharing knowledge with the broader community.
```

<div className="grid--3-col">

<Card
    title="Writing contributions"

>     body="Learn how to share and grow the collective written knowledge of the dbt Community through guides and articles."

Reviewer 3: ```suggestion
    body="Learn how to share and grow the collective knowledge of the dbt Community through blogs, guides, and documentation."
```

    link="community/contributing/contributing-writing" icon="writing"
/>

<Card
    title="Coding contributions"

>     body="The dbt Community supports a wide variety of open source projects, and OSS software is at the heart of everything we do. Learn how to get involved with coding on projects in the dbt ecoystem."

Reviewer 3: ```suggestion
    body="The dbt Community supports a wide variety of open source projects, and OSS software is at the heart of everything we do. Learn how to get involved with projects in the dbt ecosystem."
```

    link="community/contributing/contributing-coding"
    icon="folder"
/>

<Card
    title="Online community building"

>     body="Getting involved in the Slack and Discourse commmunities is one of the best entry points for contributing to the dbt Community. Learn how to share and learn from community knowledge online."

Reviewer 3: ```suggestion
    body="Getting involved in the Slack and Discourse commmunities is one of the best entry points for contributing to the dbt Community. Share your knowledge and learn from the community online."
```

>     body="Getting involved in the dbt Slack commmunity or Community Forum is one of the best entry points for contributing to the dbt Community. Learn how to share and learn from community knowledge online."

Reviewer 3: ```suggestion
    body="Getting involved in the dbt Community  Forum or Slack is one of the best entry points for contributing. Share your knowledge and learn from others."
```

    link="community/contributing/contributing-online-community"
    icon="discussion"
/>

<Card
    title="Realtime event participation"

>     body="Want to attend or even speak at a Meetup or conference? This guide will walk you through how."

Reviewer 3: ```suggestion
    body="Want to speak at a Meetup or conference? Learn how to get involved and check out best practices for crafting a talk that everyone will remember." 
```
